recipes/r-forestfloor/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+if [[ $target_platform =~ linux.* ]] || [[ $target_platform == win-32 ]] || [[ $target_platform == win-64 ]] || [[ $target_platform == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+
+  DISPLAY=${DISPLAY:-:0} $R CMD INSTALL --build .
```

Comment:
Thanks for the updates! I meant the DISPLAY part here as well, still needed?

Reply:
@cbrueffer no :-), is not needed. Removed.